components/navigation/learningItems.js

```diff
@@ -1,11 +1,13 @@
 import IconRocket from '../icons/Rocket'
 import IconGradCap from '../icons/GradCap'
 import IconPlant from '../icons/Plant'
+import IconGuide from '../icons/Guide'
 import IconPaper from '../icons/Paper'
 
 export default [
   { href: '/docs/concepts', icon: IconRocket, className: 'bg-secondary-200', title: 'Concepts', description: 'Our Concepts section defines the concepts of AsyncAPI features and capabilities.' },
-  { href: '/docs/tutorials', icon: IconGradCap, className: 'bg-pink-100', title: 'Tutorials', description: 'Our Tutorials section teaches beginner processes with AsyncAPI by doing. ' },
+  { href: '/docs/tutorials', icon: IconGradCap, className: 'bg-pink-100', title: 'Tutorials', description: 'Our Tutorials section teaches beginner processes with AsyncAPI, guiding you from Point A to Point B.' },
   { href: '/docs/tools', icon: IconPlant, className: 'bg-green-200', title: 'Tools', description: 'Our Tools section documents the AsyncAPI tools ecosystem.' },
+  { href: '/docs/guides', icon: IconGuide, className: 'bg-primary-200', title: 'Guides', description: 'Our Guides section teaches AsyncAPI's capabilities at a high level.' },
```

Comment:
@magicmatatjahu Is it possible this broke because I added an apostrophe to `AsyncAPI's`? 🧐

Reply:
Exactly, that's what's making an error. Just make sure you make the parent string under double quotes.
```suggestion
  { href: '/docs/guides', icon: IconGuide, className: 'bg-primary-200', title: 'Guides', description: "Our Guides section teaches AsyncAPI's capabilities at a high level." },
```